tests/x11/xfce_lightdm_logout_login.pm

```diff
@@ -25,7 +25,7 @@ sub run {
     mouse_set(100, 100);
     for (1 .. 4) {
         mouse_hide;
-        check_screen('mouse-cursor', 1) || return;
+        check_screen('mouse-cursor', 0) || return;
```

Comment:
I guess it may break test suite. 1 second is not same as 0. So I thought we want to start with explicit timeout value and not changing behavior and then getting there...

Reply:
reverted